tests/framework/hooks/test_hook_pipeline_ml.py

```diff
@@ -575,3 +629,73 @@ def test_mlflow_hook_save_pipeline_ml_with_artifact_path(
         )
         # the real test is that the model is loaded without error
         assert trained_model is not None
+
+
+@pytest.mark.parametrize(
+    "namespace",
+    (["a", "b"]),
```

Comment:
I don't think it is useful to test 2 different values of namespace? This does not add coverage or give more confidence, does it?

Reply:
Removed the parameterize part